config/Makefile.mingw64

```diff
@@ -106,6 +106,7 @@ FORCE_SAFE_STRING=false
 DEFAULT_SAFE_STRING=true
 WINDOWS_UNICODE=1
 AFL_INSTRUMENT=false
+AWK=gawk
```

Comment:
Is there a reason to use gawk, not awk under Cygwin?

Reply:
Yes - `awk` is a symlink, so can't be used in `-pp`

Reply:
See commit https://github.com/ocaml/ocaml/pull/1010/commits/8c0fc12b56fc0523bc82b50694e0b2c68e7bce60